### PR TITLE
Add private evidence autosaving into inventories/autosave.ini

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Private evidence autosaves to `inventories/autosave.ini` when "evidence ok" is clicked.
It is also automatically loaded when courtroom is constructed.

### CODE REVIEW

**Comment by stonedDiscord** [comment]
should it save before or after the sync?
i'd say after

File: src/evidence.cpp [hunk 1]
```diff
@@ -757,29 +758,57 @@ void Courtroom::on_evidence_load_clicked()
...
+
+void Courtroom::evidence_save(QString filename)
+{
+  // "Inventories" dir keeps our private evidence data
+  if (!dir_exists("inventories")) {
+    // Create one if it doesn't yet exist
+    QDir("inventories").mkdir("inventories");
+  }
+
+  QSettings inventory(filename, QSettings::IniFormat);
```
Comment (oldmud0) [inline src/evidence.cpp:802]:
```suggestion
  QSettings inventory(filename, QSettings::IniFormat);
  inventory.setIniCodec("UTF-8");
```

**Review by oldmud0 — CHANGES_REQUESTED** [review]

File: src/evidence.cpp (hunk 1 above)
Comment (Salanto) [reply src/evidence.cpp:802]: It does. This is why the fav list was broken when special characters was used.

**Comment by stonedDiscord** [comment]
you got a merge conflict bro

File: src/evidence.cpp [hunk 2]
```diff
@@ -616,6 +617,11 @@ void Courtroom::on_evidence_ok_clicked()
       set_evidence_page();
     }
   }
+
+  // Autosave private evidence
+  if (!current_evidence_global) {
+    evidence_save("inventories/autosave.ini");
```
Comment (Salanto) [inline src/evidence.cpp:623]: Why is this in a folder, with a plural if we can only have one, especially when the autosave file has such a non-descript name about its content. This just screams "I want to add more of this bs but sneak it into the file structure now so people won't stop me later."

Comment (Crystalwarrior) [reply src/evidence.cpp:632]: Because "Inventoriers" is the default folder it tries to access when you press the "Save" button @Salanto

Comment (Salanto) [reply src/evidence.cpp:635]: Cringe default path. Fine.

**Comment by Salanto** [comment]
Personal thing, but I'd remove the automatic loading during courtroom construction as it may cause the private evidence to be cluttered with old autosaves which forces people to remove those first or delete the backup file manually.

**Review by stonedDiscord — APPROVED** [review]

**Pull request merged**